controllers/storagecluster/reconcile.go

```diff
@@ -318,7 +318,10 @@ func (r *StorageClusterReconciler) reconcilePhases(
 			}
 		}
 
-		if err := r.reconcileUninstallAnnotations(instance); err != nil {
+		if scWasUpdated, err := r.reconcileUninstallAnnotations(instance); err != nil {
+			return reconcile.Result{}, err
+		} else if scWasUpdated {
```

Comment:
Are you sure the `scWasUpdated` is in scope here?

Reply:
Fair question... can you extend the unit tes you touched to check the boolean?

Reply:
yes

Reply:
@jarrpa done